src/axom/core/Array.hpp

```diff
@@ -224,7 +224,9 @@ class Array : public ArrayBase<T, DIM>
    * \post size() == num_elements
    * \post getResizeRatio() == DEFAULT_RESIZE_RATIO
    */
-  template <typename... Args>
+  template <IndexType SFINAE = DIM,
+            typename std::enable_if<SFINAE != 1>::type* = nullptr,
+            typename... Args>
```

Comment:
I had a slightly different fix for this in my current WIP branch that checks if the first argument in the parameter pack `is_integral` - though it would probably be more thorough to check the whole pack and/or add this conditional.  What was the context for this change?

Reply:
Yeah, this was mostly to get it building on CI -- I'll defer to whatever fix is in your branch :P

Basically, the issue I was running into was that on the CI job for gcc 8.4.0, there was some ambiguity with a call to the `Array(num_elems, capacity, allocator_id)` constructor - it seems like if any of the arguments have to be implicitly-converted, it will prefer the variadic size constructor even if the Array instance is a single-dimension array.